upgrade/money-class.md

```diff
@@ -449,7 +449,7 @@ As an example, see the change that was required in `@ShopsysShop/Front/Content/C
 +       {% for index, cartItem in cartItems %}
 +           {% set cartItemPrice = cartItemPrices[index] %}
 +           {% set cartItemDiscount = cartItemDiscounts[index] %}
-                <tr class="table-cart__row js-cart-item">
```

Comment:
this change should not be present (in the version, where this upgrade notes are present, the `test-cart-item` will not work)

Reply:
26b54de3876ccf66bd1593aeca8225bc9454c442